src/ordered-dict.ts

```diff
@@ -102,6 +110,12 @@ export class OrderedDict<T> {
     return OrderedDict.fromProplist(items);
   }
 
+  public forEach<U>(callback: (value: T, key: string, index: number) => U) {
```

Comment:
forEach, by definition, should not return anything, so instead of <U> you don't want a return type at all (void)